security/cmd/node_agent_k8s/main.go

```diff
@@ -83,6 +83,10 @@ const (
 	vaultTLSRootCert     = "VAULT_TLS_ROOT_CERT"
 	vaultTLSRootCertFlag = "vaultTLSRootCert"
 
+	// The environmental variable name for Spire TLS bootstrap certificate.
```

Comment:
For better readability, can you add a comment for the purpose of Spire TLS bootstrap certificate (e.g., when this certificate is used and for which communicating party it is used)?